### ISSUE TITLE
how to bind 'format document' to different command palette

### ISSUE THREAD

**Issue by SuperManEver** [esbenp/prettier-vscode#170]
is it possible to setup different command palette to 'format document' because ctrl+alt+p for me is attached to different command ?

**Comment by CiGit** [comment]
I don't get your question exactly. But you can define shortcuts for every actions (keyboard shortcuts)

**Comment by SuperManEver** [comment]
@CiGit ok, sorry. Let me try again. So, I installed `prettier` as CLI tool globally and I added `prettier` as extension to vs code. I'am confident that it works because I configured it to format my code on file save. 

what I'd like to have is to be able to trigger code format on demand rather than on file save and also I'd like to apply it not to whole file but may be only for selected part.

the only thing I was able to achieve is to set 
```
{
    "key": "shift+alt+f",
    "command": "prettier.open-output"
}
```

I'd like to do similar thing for code formatting, but I don't know which `prettier` command to trigger.

I hope this makes more sense :)

**Comment by CiGit** [comment]
Get it :-)

You found our hidden action! 
those you are searching for are:
```json
    {
        "key": "shift+alt+f",
        "command": "editor.action.formatDocument",
        "when": "editorHasDocumentFormattingProvider && editorTextFocus && !editorReadonly"
    },
    {
        "key": "ctrl+k ctrl+f",
        "command": "editor.action.formatSelection",
        "when": "editorHasDocumentSelectionFormattingProvider && editorHasSelection && editorTextFocus && !editorReadonly"
    }
```
Those are from my editor and are the default ones.

**Comment by SuperManEver** [comment]
@CiGit works wonderfully. Thank you for help

**Comment by CiGit** [comment]
Great you had your answer. Closing
